test/DGOV.ts

```diff
@@ -1,84 +1,64 @@
+        /*
+            MS = 1000000
+            Air.S = 250000
+            Allo.S = 100000
+            CS = TS - Air.S - Allo.S = 650000
+        */
+        expect(web3.utils.toNumber(await dgovObj.getMaxCollateralisedSupply())).to.equal(650000);
     });
 
-
-    it('able to mint the airdroped supply', async () => {
-
-        await dgovObj.mintAirdropSupply(user2, web3.utils.toNumber(100), {from: airdropAddress});
-        expect(web3.utils.toNumber(await dgovObj.getAirdropBalance(user2, {from: user2}))).to.equal(web3.utils.toNumber(100));
-
+    it('Max Allocated Supply Check', async () => {
+        //10% of MS = 100000
+        expect(web3.utils.toNumber(await dgovObj.getMaxAllocatedSupply())).to.equal(100000);
     });
 
-    it('total supply is sum of the airdropAddress + collateral + allocated supply ', async () => {
-        await dgovObj.mintAirdropSupply(user2, web3.utils.toNumber(100), {from: airdropAddress});
-        await dgovObj.mintCollateralisedSupply(user2, web3.utils.toNumber(200), {from: bankAddress});
-        await dgovObj.mintAllocatedSupply(user2, web3.utils.toNumber(300), {from: governance});
-        console.log('total balance of user2:'+  web3.utils.toNumber(await dgovObj.getTotalBalance(user2)))
-       // expect(web3.utils.toNumber(await dgovObj.getTotalBalance(user2))).to.equal(web3.utils.toNumber(600));
+    it('Mint Collateralized Supply Check', async () => {
+        await dgovObj.mintCollateralisedSupply(user1, 650000, { from: bankAddress });
+        expect(web3.utils.toNumber(await dgovObj.balanceOf(user1))).to.equal(web3.utils.toNumber(await dgovObj.getTotalCollateralisedSupply()));
     });
 
-
-    it('mint allocated supply should work for higher values then airdrop supply  ', async () => {
-        dgovObj.setMaxSupply(web3.utils.toNumber(10000000), {from:governance});
-        await dgovObj.mintAirdropSupply(user2, web3.utils.toNumber(100), {from: airdropAddress});
-        await dgovObj.mintAirdropSupply(User1, web3.utils.toNumber(50), {from: airdropAddress});
-        await dgovObj.(User1, web3.utils.toNumber(50), {from: airdropAddress});
-        expect(web3.utils.toNumber(await dgovObj.getLockedBalance(user2))).to.equal(web3.utils.toNumber(288));
-    })
-
-
-    it('mint collateralised supply should work for higher values than airdrop supply ', async () => {
-        dgovObj.setMaxSupply(web3.utils.toNumber(10000000), {from:governance});
-        await dgovObj.mintAirdropSupply(user2, web3.utils.toNumber(100), {from: airdropAddress});
-        await dgovObj.mintAirdropSupply(User1, web3.utils.toNumber(100), {from: airdropAddress});
-        await dgovObj.mintCollateralisedSupply(user2, web3.utils.toNumber(1000), {from: bankAddress});
-        expect(web3.utils.toNumber(await dgovObj.getCollateralisedBalance(user2))).to.equal(web3.utils.toNumber(1000));
+    it('Mint Allocated Supply Check', async () => {
+        await dgovObj.mintAllocatedSupply(user2, 100000, { from: governanceAddress });
```

Comment:
thanks , but i have an query : 

do we need to repeat the unit tests for the functionality that are similar in debond and DBIT, or we put the common tests in the debondToken.ts  and then extend to only the tests of dgov that  are needed to be deployed.